src/components/dev-grid-utils.js

```diff
@@ -56,4 +56,11 @@ export const profiles = [
     photo: null,
     siteUrl: null,
   },
+  {
+    githubUrl: "https://github.com/",
```

Comment:
Don't forget to add your GitHub url.

Reply:
Ohh yes, sure, just added the username. Thanks!